.github/workflows/ci-pipeline.yml

```diff
@@ -27,6 +27,9 @@ env:
   CONTAINER_REGISTRY: ghcr.io
   LOCAL_REGISTRY: registry.localhost:5000
 
+  CARAML_CHARTS_REPOSITORY: romanwozniak/helm-charts
```

Comment:
Currently, the CI pipeline of [`gojek/mlp`](https://github.com/gojek/mlp) depends on the chart from [`caraml-dev/helm-charts`](https://github.com/caraml-dev/helm-charts) for running E2E tests. 

At the same time, [`caraml-dev/helm-charts`](https://github.com/caraml-dev/helm-charts) depends on the build artifacts (i.e. docker image) from [`gojek/mlp`](https://github.com/gojek/mlp), because CI in `helm-charts` actually installs the charts to test them.

Since this refactoring introduces breaking changes, the docker image built from this PR is incompatible with any existing `mlp` charts (hence it's using the chart from the unmerged PR), but also, the new version of the chart is not compatible with any published version of the app (hence CI for https://github.com/caraml-dev/helm-charts/pull/98 is red).

We discussed some options for decoupling the app's CI from the chart internally over Slack and I'm waiting for the Platform folks to put the proposal together. But in the meantime, I'll need to merge this PR first (with the feature branch of the chart repo used in CI) and revert it to use the main chart's repo/branch in a separate PR later.

cc @maniteja6799 @mbruner